### ISSUE TITLE
InnerSource Commons or InnerSource Commons Foundation

### ISSUE THREAD

**Issue by MaineC** [InnerSourceCommons/InnerSourceLearningPath#223]
In existing texts occasionally we are referring to the InnerSource Commons. Should we adjust the texts to talk about the InnerSource Commons Foundation?

**Comment by rrrutledge** [comment]
I don't think so.  "Commons" already indicates it's some kind of group thing.  "Foundation" was just a legal requirement for incorporating.

I think that the real name of "Nike" is "Nike, Inc" but no one calls it that.
